lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -192,4 +192,8 @@ export abstract class ClientApplication {
             this.storage.getCache() as InMemoryCache
         );
     }
+
+    getAllAccounts(): IAccount[] {
+        return AuthorizationCodeClient.getAllAccounts(this.storage);
```

Comment:
This is confusing. What if the user is not doing authorization code flow?

Reply:
Doesn't matter, you can call it off any client that extends the base client.